packages/babel-plugin-proposal-record-and-tuple/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
why does this need a different license than most of the other packages, which are MIT?